templates/download/server/step2.html

```diff
@@ -1,53 +1,216 @@
+      <hr>
+    </div>
+  </div>
+  <div class="row u-equal-height" id="releases">
+    <div class="col-6">
+      <h2>Alternative releases</h2>
+      <h3>
+        Ubuntu Server {{ releases.latest.full_version }}
+      </h3>
+      <p>
+        The latest version of Ubuntu Server, including nine months of security and maintenance updates, until {{ releases.latest.eol }}.
+      </p>
+      <form action="/download/server" method="post" style="display: inline">
+        <input type="hidden" name="version" value="{{ releases.latest.full_version }}">
+        <input type="hidden" name="architecture" value="amd64">
+        <input type="hidden" name="next-step" value="download">
+        <button type="submit" class="p-button--positive" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Server download', 'eventAction' : 'Select option', 'eventLabel' : 'Manual install - latest', 'eventValue' : undefined });">Download Ubuntu Server {{ releases.latest.full_version }}</button>
+      </form>
+
+      <p>
+        <a href="https://www.brighttalk.com/webcast/6793/448578?utm_source=brighttalk-portal&utm_medium=web&utm_content=ubuntu%2020.10&utm_campaign=webcasts-search-results-feed" class="p-link--external">
+          Watch the webinar - "What's new in Ubuntu 20.10?"
+        </a>
+      </p>
+
+      <p>
+        <a href="https://wiki.ubuntu.com/{{ releases.latest.slug }}/ReleaseNotes" class="p-link--external">
+          Download Ubuntu Server {{ releases.latest.short_version }} release notes
+        </a>
+      </p>
+    </div>
+    <div class="col-4 col-start-large-8 u-hide--small u-align--center u-vertically-center">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/fe951eda-20.10_Groovy+Gorilla_RPi_Sketch.svg",
+          alt="",
+          height="237",
+          width="153",
+          hi_def=True,
+          loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+  {% endif %}
+</section>
+
+<section class="p-strip--light u-no-padding--bottom">
+  <div class="row">
+    <div class="col-10">
+      <h2>Alternative downloads</h2>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-4 p-card">
+      <h3 class="p-card__title">BitTorrents</h3>
+      <p>BitTorrent sometimes enables higher download speeds and more reliable downloads of large files.</p>
+      <ul class="p-list">
+        <li class="p-list__item">
+          <a class="download-torrent" href="https://releases.ubuntu.com/{{ releases.lts.short_version }}/ubuntu-{{ releases.lts.full_version }}-live-server-amd64.iso.torrent">
+          Ubuntu Server {{ releases.lts.full_version }} LTS&nbsp;&rsaquo;
+        </a>
+        </li>
+        <li class="p-list__item">
+          <a class="download-torrent" href="https://releases.ubuntu.com/{{ releases.latest.short_version }}/ubuntu-{{ releases.latest.full_version }}-live-server-amd64.iso.torrent">
+          Ubuntu Server {{ releases.latest.full_version }}&nbsp;&rsaquo;
```

Comment:
These are inconsistent with the eight other “›” links on this page, which go to Web pages; and inconsistent with the following card, where a direct download is styled as a button (with no visible hint that it actually needs to be one). These should be buttons too.

Reply:
Done

Reply:
I think this looks much better.